.github/workflows/benchmark_master.yml

```diff
@@ -0,0 +1,77 @@
+# Workflow to trigger/schedule benchmarking, cleaning, aggregating on master branch only and storing results in S3
+name: Benchmark Master Branch
+on:
+  push:
+    branches: ["master"]
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
Can we change this to be around 2am in the morning PST. By default, this is UTC time